### DISCUSSION TITLE
Radio improvement - Primary transmit and receive with secondary receive only channel + encryption

### DISCUSSION THREAD

**Discussion by xSideburns** [FakeFishGames/Barotrauma#6147] [Ideas and suggestions]
It might be useful to have the ability to receive and transmit on a primary channel, but also listen in / receive on a second channel (perhaps split to left and right ear for each channel / a bit like a guard channel on military aircraft). e.g. if you were the captain you could be on transmit & receive for main channel onboard vessel but receive only on a secondary channel being used by an away team to listen in and be aware of things. Or alternatively be able to assign teams channels (Command channel, engineering channel, security channel while having an emergency guard channel also). UI would need to be updated to perhaps colour code or indicate T+R and R only selected channels. On channel selection perhaps a single click for T+R channel and double click for R only. Finally headset perhaps uses twice the power when in T+R and R dual channel mode?

Also password protected / encrypted channels might also be a nice idea, with a distorted sound coming out if you don't have the key, would probably be a major assist for traitor teams but also a give away if someone notices the channel being used.

Or these suggestions might make the game too easy and you want to retain the single channel radio?

**Comment by Darmdos** [comment]
As requested on Discord, here's my similar idea which could serve as an option / alternative / addition to Sideburns' idea:

"It would be nice to be able to listen and broadcast to different radio channels simultaneously. Could be a different headset or a "master radio" which has this ability. Could also be definable to certain roles (idcard). This ability would allow, for example captains, to broadcast over everyone else to announce something important."
